xptifw/basic_test/CMakeLists.txt

```diff
@@ -1,22 +1,24 @@
-cmake_minimum_required(VERSION 2.8.9)
-project (run_test)
-
 file(GLOB SOURCES *.cpp *.hpp)
 include_directories(${XPTIFW_DIR}/include)
 include_directories(${XPTI_DIR}/include)
 
-remove_definitions(-DXPTI_STATIC_LIBRARY)
-add_definitions(-DXPTI_API_EXPORTS -g -O3)
-add_executable(run_test ${SOURCES})
-add_dependencies(run_test xptifw)
-target_link_libraries(run_test PRIVATE xptifw)
-if(UNIX)
-  target_link_libraries(run_test PRIVATE dl)
-endif()
+add_executable(XPTIFWBasicTests ${SOURCES})
+target_link_libraries(XPTIFWBasicTests PRIVATE xptifw ${CMAKE_DL_LIBS})
 
-if (XPTI_ENABLE_TBB)
-  target_link_libraries(run_test PRIVATE tbb)
+if (UNIX)
+  target_compile_definitions(XPTIFWBasicTests PRIVATE -gline-tables-only -O3)
```

Comment:
What's relying on this fine-tuned choice of options: "-gline-tables-only -O3"?

Reply:
These tests try to model real application and are meant to be profiled often. This is just a convenient set of options for perf tool.